src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -611,6 +620,22 @@ public static void GetFullPath_Windows_ValidExtendedPaths(string path)
             }
         }
 
+        [PlatformSpecific(TestPlatforms.Windows)]  // Tests Windows-specific paths
+        [Theory]
+        [InlineData(@"\\.\UNC\")]
+        [InlineData(@"\\.\UNC\server7")]
```

Comment:
Unless you are *absolutely* sure you are doing nothing that will hit the file system. Path.GetFullPath will, in some cases, hit the disk- notably if it has to expand short file names (which currently is gated on containing a `~`)